wger/settings_global.py

```diff
@@ -35,13 +35,24 @@
 ROOT_URLCONF = 'wger.urls'
 WSGI_APPLICATION = 'wger.wsgi.application'
 
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = '785741301455-0fpulhilcg0u6urkd0ifcnctr292oa3m.apps.googleusercontent.com' #CLient Key
```

Comment:
It is a good practice to store sensitive information in environment variables. I think the social keys should also live in the environment variables too.

Reply:
Thanks @Rodgers-M I'll change it.